README.md

```diff
@@ -26,8 +26,13 @@ Please checkout [ML on GCP](https://github.com/GoogleCloudPlatform/ml-on-gcp) gu
 * [Reddit](reddit_tft) - Predict the score of a Reddit thread using a wide and deep model
 * [Cifar10](https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator) - Classify image content (training on multiple GPUs)
 
-### TPU (Contrib)
-* [Resnet TPU](tpu)
+### TPU
+* [MNIST on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/mnist)
+* [ResNet-50 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/resnet)
+* [Inception on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception)
+* [Inception V3 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception-v3-advanced)
```

Comment:
Let's change this line to:

```
[Advanced walkthrough of Inception v3 on Cloud TPU](https://cloud.google.com/tpu/docs/tutorials/inception-v3-advanced)
```

My reasoning is that the straight "inception" tutorial  (mentioned on the line above) covers Inception v3 and v4. So, it may mislead readers if the next line mentions v3 explicitly without including the fact that the second tutorial is an advanced walkthrough.

Reply:
ptal